datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1467,6 +1467,14 @@ CREATE VIEW protocol_upgrade_proposals_current AS (
       FROM protocol_upgrade_proposals
   ORDER BY upgrade_block_height, vega_release_tag, vega_time DESC);
 
+CREATE TABLE IF NOT EXISTS core_snapshots(
+    block_height BIGINT NOT NULL,
+    block_hash bytea not null,
+    tx_hash bytea not null,
```

Comment:
isn't this going to be the block hash anyways seeing that the snapshots are not triggered on a transaction but end block or commit (can't remember)

Reply:
it is, and we don't strictly need it here, I just kept it for consistency with the other tables. can drop it